### DISCUSSION TITLE
🔗 Add Fade as Transition Effect in the Nested Carousel

### DISCUSSION THREAD

**Discussion by Tacodiva** [elementor/elementor#24028] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want to use the "slides" element with a fading transition, but I don't want the content of each slide to be fixed to only a title, text and one button. I have tried using the nested Carousel widget, but it does not seem to support fading between content.

### Describe the solution you'd like

It would be nice if the existing "Slides" widget was changed to allow nested elements, like the Carousel.

### Describe alternatives you've considered

This could also be solved by:

- Adding a fade transition to the existing Carousel element. To be honest, I'm not familiar enough with the Carousel element (or Elementor in general) to know if this is a better idea than just changing the Slides widget. I'm not sure how the fade would work if more than one slide was on display. 

 - Rather than modifying the existing "Slides" element, adding some new element which had the functionality of Slides but supported nested content.

**Comment by sara-kariv** [comment]
++++++++++++++++++++++1

**Comment by charlesb9** [comment]
+1 The Nested Carousel widget currently uses Swiper under the hood, but the widget UI does not expose Swiper’s effect option (e.g. fade, cube, coverflow, etc.). Many hero/header or other use-cases require a simple cross-fade transition rather than horizontal sliding.
